bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,330 @@
+  "amount": "1000000ubbn",
+  "moniker": "my-validator",
+  "commission-rate": "0.10",
+  "commission-max-rate": "0.20",
+  "commission-max-change-rate": "0.01",
+  "min-self-delegation": "1"
+}
+EOF
+```
+
+Parameters:
+- `pubkey`: Your validator's public key
+- `amount`: Initial self-delegation amount
+- `moniker`: Your validator's name/identifier
+- `commission-rate`: Current commission rate 
+- `commission-max-rate`: Maximum commission rate allowed
+- `commission-max-change-rate`: Maximum daily commission change rate
+- `min-self-delegation`: Minimum amount you must keep self-delegated
+
+If you prefer to add this manually or are having issues, another option is to 
+create a `validator.json` file and then paste the above json into it but remember
+to replace the `pubkey` value with your actual validator public key.
+
+## 5. Creating a Validator
+
+> ⚠️ **Important**: You will need a funded account for this step
+
+Unlike traditional Cosmos SDK chains that use the `staking` module, 
+Babylon uses the [`checkpointing`](https://docs.babylonlabs.io/docs/developer-guides/modules/checkpointing) module for validator creation and management.
+
+The creation process requires your previously generated BLS key, 
+which should be located at `<path>/config/priv_validator_key.json`, 
+where `<path>` is the `--home` directory you specified when setting up your node.
+
+To create your validator, run the following command:
+
+```shell 
+babylond tx checkpointing create-validator \
+    ./<home-path>/config/validator.json \
+    --chain-id bbn-test-5 \
+    --gas "auto" \
+    --gas-adjustment 1.5 \
+    --gas-prices "0.005ubbn" \
+    --from <your-key-name>
+    --keyring-backend <keyring-backend>
+```
+
+Parameters:
+- `--chain-id`: The network identifier
+- `--gas`: Set to "auto" to automatically calculate the gas needed
+- `--gas-adjustment`: A multiplier for the estimated gas 
+- `--gas-prices`: Transaction fee in ubbn per unit of gas
+- `--from`: Your key name or address that will sign and pay for this transaction
+
+Upon successful creation, you'll be asked to approve the transaction. 
+Once approved, you'll receive a transaction hash and your validator
+operator address 
+(e.g., `bbnvaloper1qh8444k43spt6m8ernm8phxr332k85teavxmuq`).
+
+### 5.1 Verifying Validator Setup
+
+To verify your validator setup, you can use the following steps:
+
+First, get your validator's operator address using your Babylon address: 
+
+```shell
+babylond keys show <your-key-name> --address --bech val --home <path> --keyring-backend <keyring-backend>
+```
+
+For example, for the address we used above is `bbn1qh8444k43spt6m8ernm8phxr332k85teavxmuq`, 
+the operator address is `bbnvaloper1qh8444k43spt6m8ernm8phxr332k85teavxmuq`. 
+
+Next, inspect your validator's details: 
+
+```shell 
+babylond query staking validator <validator-operator-address>
+```
+
+The output should return the selected validator's configuration.
```

Comment:
Can we verify that this command immediately returns the details and the details are not presented at the start of the next epoch? If the details are only presented at start of the next epoch, we should give the reader a brief reminder on epochs and the need to wait (and how long). We could tell them to look by transaction hash initially to verify success, and later check their validator registration.

Otherwise, if this returns results immediately, then all good.